index.js

```diff
@@ -1,120 +1,55 @@
 #!/usr/bin/env node
-var fs = require("fs");
-var path = require("path");
-var indent = require("./lib/indent");
-var prepare = require("./lib/prepare");
-var dependencies = require("./lib/dependencies");
-var link = require("./lib/link");
-var list = require("./lib/list");
-var github = require("./lib/github");
-
-var argv = require('yargs').argv
-
-var working_directory = process.cwd();
-
-if (argv.l) {
-  list(working_directory);
-  process.exit();
-}
-
-var organization = "trufflesuite";
-
-var prepared = {};
-var packages = argv._;
-
-var baseBranch = "*";
-
-if (packages.length == 0) {
-  packages = ["truffle:" + baseBranch];
-}
-
-// For any package that doesn't specify a branch selector, specify it.
-packages = packages.map(function(package) {
-  if (package.indexOf("@") < 0 && package.indexOf(":") < 0) {
-    package = package + ":" + baseBranch;
-  }
-  return package;
-});
-
-var branchOverrides = {};
-
-// Record all 1-off branch overrides ("@" selector)
-packages.forEach(function(package) {
-  if (package.indexOf("@") >= 0) {
-    var pieces = package.split("@")
-    branchOverrides[pieces[0]] = pieces[1];
-  }
-});
-
-packages = packages.reverse();
-
-console.log("Getting package list from Github...");
-console.log("");
-
-github.repositories(organization).then(function(response) {
-  var repositories = response.data.map(function(repo) {
-    return repo.name;
-  });
-
-  while (packages.length > 0) {
-    var currentPackage = packages.shift();
-
-    var match = currentPackage.match(/(.*)(@|:)(.*)/);
-
-    var currentPackageName = match[1];
-    var selector = match[2];
-    var packageBranch = branchOverrides[currentPackageName] || match[3];
-    var dependencyBranch = baseBranch;
-
-    if (selector == ":") {
-      dependencyBranch = packageBranch;
+var link = require("./commands/link");
+var list = require("./commands/list");
+var use = require("./commands/use");
+var switchCmd = require("./commands/switch");
+
+require('yargs')
+  .command('list', 'List managed packages',
+    function() {},
+    function(argv) {
+      list({ inDir: argv.dir });
```

Comment:
What do you think about `workon` ala [virtualenvwrapper](http://virtualenvwrapper.readthedocs.io/en/latest/command_ref.html) ?